selendroid-server/src/main/java/io/selendroid/server/model/DefaultAppiumUiAutomatorDriver.java

```diff
@@ -0,0 +1,288 @@
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.selendroid.server.model;
+
+
+import android.support.test.uiautomator.UiDevice;
+import android.test.InstrumentationTestCase;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.math.BigInteger;
+import java.security.SecureRandom;
+import java.util.List;
+import java.util.Set;
+
+import io.selendroid.server.ServerInstrumentation;
+import io.selendroid.server.android.InstrumentedKeySender;
+import io.selendroid.server.android.KeySender;
+import io.selendroid.server.android.internal.Dimension;
+import io.selendroid.server.common.exceptions.SelendroidException;
+import io.selendroid.server.util.SelendroidLogger;
+
+//import android.support.test.uiautomator.UiDevice;
+
+public class DefaultAppiumUiAutomatorDriver implements SelendroidDriver {
+
+    public static final String BROWSER_NAME = "browserName";
+    public static final String PLATFORM = "platform";
+    public static final String SUPPORTS_JAVASCRIPT = "javascriptEnabled";
+    public static final String TAKES_SCREENSHOT = "takesScreenshot";
+    public static final String VERSION = "version";
+    public static final String SUPPORTS_ALERTS = "handlesAlerts";
+    public static final String ROTATABLE = "rotatable";
+    public static final String ACCEPT_SSL_CERTS = "acceptSslCerts";
+    public static final String SUPPORTS_NETWORK_CONNECTION = "networkConnectionEnabled";
+    private static UiDevice device;
+    private ServerInstrumentation serverInstrumentation = null;
+    private KeySender keySender = null;
+    private Session session = null;
+
+    public DefaultAppiumUiAutomatorDriver(ServerInstrumentation serverInstrumentation) {
+        this.keySender = new InstrumentedKeySender(serverInstrumentation);
+        ;
+        this.serverInstrumentation = serverInstrumentation;
+    }
+
+    @Override
+    public AndroidElement findElement(By by) {
+        return null;
+    }
+
+    @Override
+    public List<AndroidElement> findElements(By by) {
+        return null;
+    }
+
+    @Override
+    public String getCurrentUrl() {
+        return null;
+    }
+
+    @Override
+    public Session getSession() {
+        return session;
+    }
+
+    @Override
+    public JSONObject getSessionCapabilities(String sessionId) {
+        SelendroidLogger.info("session: " + sessionId);
+
+        JSONObject copy;
+        try {
+            JSONObject capabilities = session.getCapabilities();
+            if (capabilities != null) {
+                copy = new JSONObject(capabilities.toString());
+            } else {
+                copy = new JSONObject();
+            }
+            copy.put(TAKES_SCREENSHOT, true);
+            copy.put(BROWSER_NAME, "selendroid");
+            copy.put("automationName", "UiAutomator");
```

Comment:
These string keys should be refactored to be `public static final String` constants just like `TAKES_SCREENSHOT`, `BROWSER_NAME`, and the others.